app/assets/javascripts/views/story_view.js

```diff
@@ -883,6 +884,31 @@ module.exports = FormView.extend({
     $('.popover').remove();
   },
 
+  backLoggedRelease: function() {
+    let backlogged = false;
+    if(typeof this.model.collection.project.iterations !== 'undefined') {
+      this.model.collection.project.iterations.forEach((iteration) => {
+        iteration.stories().forEach((story) => {
+          if (story.id === this.model.attributes.id && this.model.attributes.release_date) {
+            let iteration_date = new Date(iteration.startDate());
+            let release_date = new Date(this.model.attributes.release_date);
+            backlogged = iteration_date > release_date;
+          }
+        })
+      });
+      return backlogged;
+    }
+  },
+
+  handleBackLoggedRelease: function() {
+    if(this.backLoggedRelease()){
+      this.$el.addClass('backlogged-release');
```

Comment:
You could simplify this conditional using `toggleClass`, like:
```js
this.$el.toggleClass('backlogged-release', this.backLoggedRelease());
if (this.backLoggedRelease()) {
  this.$el.attr('title', I18n.t('story.warnings.backlogged_release'));
}
```

Reply:
Also, avoid using else, most of the time you can achieve the same logic using just `if` and `return`